src/crepe/api/Transform.h

```diff
@@ -2,12 +2,83 @@
 
 #include <cstdint>
 
-#include "api/Point.h"
-
 #include "Component.h"
+#include <cstdint>
+#include <cmath>
 
 namespace crepe {
 
+//! Vector2 struct
```

Comment:
Ondanks dat Vector2 wellicht niet echt bij deze PR hoort, vind ik het niet oké om Vector2 in hetzelfde bestand te plaatsen als Transform. Transform is inmiddels volgens de code standaard en dat willen we ook graag zo houden. Vector2 mag dus in een nieuw bestand.